apps/web/src/components/VerificationCode/EnterPhoneNumber.tsx

```diff
@@ -0,0 +1,66 @@
+import { Typography } from "@litespace/ui/Typography";
+import { Animate } from "@/components/Common/Animate";
+import { useFormatMessage } from "@litespace/ui/hooks/intl";
+import { Input } from "@litespace/ui/Input";
+import { Button } from "@litespace/ui/Button";
+import { Void } from "@litespace/types";
+import { orUndefined } from "@litespace/utils";
+
+type ChoosePhoneNumberProps = {
+  phoneNumber: string | null;
+  setPhoneNumber: (phone: string) => void;
+  sendVerificationCode: Void;
+  sending: boolean;
+  close: Void;
+};
+
+export function ChoosePhoneNumber({
+  close,
+  phoneNumber,
+  sendVerificationCode,
+  sending,
+  setPhoneNumber,
+}: ChoosePhoneNumberProps) {
+  const intl = useFormatMessage();
+
+  return (
+    <Animate>
+      <Typography
+        tag="p"
+        className="text-caption mt-2 font-semibold text-natural-950"
+      >
+        {intl("notification-method.dialog.phone.description")}
+      </Typography>
+      <div className="mt-6 mb-12">
+        <Input
+          label={intl("labels.phone")}
+          onChange={(e) => setPhoneNumber(e.target.value)}
+          value={orUndefined(phoneNumber)}
+          placeholder={intl(
+            "notification-method.dialog.phone.input.placeholder"
+          )}
```

Comment:
Done't use regualr input. Use a [pattern input](https://github.com/litespace-org/litespace/blob/61a92cf1e4517cc9b31f5b27da7409502b37390e/apps/web/src/components/StudentSettings/ProfileForm.tsx#L225-L239). Also, please don't accept any input from the end user without validation! 

> [!NOTE]
> Make user of the new useForm in headless.